src/functions/template-tags/venue.php

```diff
@@ -460,8 +460,8 @@ function tribe_get_venue_details( $post_id = null ) {
 
 		$venue_details = array();
 
-		if ( $venue_name = tribe_get_venue( $post_id ) ) {
-			$venue_details['name'] = $venue_name;
+		if ( $venue_link = tribe_get_venue_link( $post_id ) ) {
+			$venue_details[ 'linked_name' ] = $venue_link;
```

Comment:
Array indexes that are strings shouldn't have the extra spacing inside the square brackets like indexes composed of variables/concatenation. Ref: [WordPress standards on spacing](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage)